### PR TITLE
addGitOPS

### PR DESCRIPTION
**Pull request by mnogosemok**

### CODE REVIEW

File: Vadim_Shishlo/04. GitOPS/READMI.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+# 04.GitOPS
```
Comment (pluhin) [inline Vadim_Shishlo/04. GitOPS/READMI.md:1]: Please correct name of the file (READM**E**.md)

**Review by pluhin — CHANGES_REQUESTED** [review]
remove white-space from folder name

**Review by mnogosemok — COMMENTED** [review]
add

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**